common/pom.xml

```diff
@@ -56,7 +56,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-checkstyle-plugin</artifactId>
         <configuration>
-          <maxAllowedViolations>114</maxAllowedViolations>
+          <logViolationsToConsole>true</logViolationsToConsole>
```

Comment:
Please refer at the parent pom. logViolationsToConsole is controlled by the user property checkstyle.console which by default is false